### DISCUSSION TITLE
Is DPX supposed to be a supported or is it normal it doesnt load ?

### DISCUSSION THREAD

**Discussion by NicTanghe** [Beep6581/RawTherapee#7217] [Q&A] [answered]
Is DPX supposed to be a supported or is it normal it doesnt load ?

**Comment by Lawrence37** [comment, marked as answer]
It is not supported. Supported file format families are:
- Many camera raw formats
- DNG
- JPEG
- TIFF
- JXL
- PNG

**Reply by NicTanghe** [reply]
plz add DPX

**Reply by Lawrence37** [reply]
Just to confirm, are you referring to [Digital Picture Exchange](https://en.wikipedia.org/wiki/Digital_Picture_Exchange)?

**Reply by NicTanghe** [reply]
y ( i guess this is the same problem as the .exr ) i wanted to do it at the time but i was not good enough a programmer at the time.

**Reply by Lawrence37** [reply]
I have created a new issue for adding support (#7301).
